bin/ch/RuntimeThreadData.cpp

```diff
@@ -25,10 +25,17 @@ RuntimeThreadData::RuntimeThreadData()
     this->hevntInitialScriptCompleted = CreateEvent(NULL, TRUE, FALSE, NULL);
     this->hevntReceivedBroadcast = CreateEvent(NULL, FALSE, FALSE, NULL);
     this->hevntShutdown = CreateEvent(NULL, TRUE, FALSE, NULL);
+    this->hSemaphore = nullptr;
+    this->hThread = nullptr;
```

Comment:
is there a reason not to do these in an initializer list, or better yet, in class initialization?

Reply:
Yeah, changed in next commit.